src/test/resources/org/prebid/server/it/test-application.properties

```diff
@@ -322,6 +322,8 @@ adapters.zeroclickfraud.enabled=true
 adapters.zeroclickfraud.endpoint=http://{{Host}}/zeroclickfraud-exchange?sid={{SourceId}}
 adapters.aax.enabled=true
 adapters.aax.endpoint=http://localhost:8090/aax-exchange
+adapters.undertone.enabled=true
```

Comment:
We prefer to have properties in alphabetical order, pls set it in appropriate place